src/utils/forwardToCheckoutSession.ts

```diff
@@ -0,0 +1,18 @@
+import { paymentServiceUrl } from "../constants";
+
+export async function forwardToCheckoutSession(
```

Comment:
i would maybe make this `createCheckoutSessionURL`, return the URL and the caller decides what to do (in this case, forward)

Reply:
f32c254